pal/src/arch/i386/activationhandlerwrapper.S

```diff
@@ -0,0 +1,31 @@
+// -------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+// -------------------------------------------------------------------------------------------------------
+
+.intel_syntax noprefix
+#include "unixasmmacros.inc"
+#include "asmconstants.h"
+
+#ifdef BIT64
```

Comment:
I was reading around here too and realized these were pulled from https://github.com/dotnet/coreclr/tree/master/src/pal/src/arch/i386. So that could be the answer. PAL did that.